executor/split_table_test.go

```diff
@@ -593,4 +606,100 @@ func TestShowTableRegion(t *testing.T) {
+	require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[0][12])
+	require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[1][12])
+	require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[2][12])
+
+	// Test scheduling info for partitioned table after split to regions
+	tk.MustExec("drop table if exists t3_scheduling")
+	tk.MustExec("create table t3_scheduling (id INT) placement policy p1 partition by range (id) (" +
+		"partition p0 values less than (100) placement policy p2," +
+		"partition p1 values less than (1000)," +
+		"partition p2 values less than (10000)" +
+		")")
+	tk.MustQuery("split partition table t3_scheduling between (0) and (10000) regions 4")
+	re = tk.MustQuery("show table t3_scheduling regions")
+	rows = re.Rows()
+	require.Len(t, rows, 12)
+	require.Len(t, rows[0], 13)
+	for i := range rows {
+		if i < 4 {
+			require.Equal(t, "LEADER_CONSTRAINTS=\"[+region=us-east-1]\" FOLLOWERS=3 FOLLOWER_CONSTRAINTS=\"[+region=us-east-2]\"", rows[i][11])
+		} else {
+			require.Equal(t, "PRIMARY_REGION=\"cn-east-1\" REGIONS=\"cn-east-1,cn-east-2\" SCHEDULE=\"EVEN\"", rows[i][11])
+		}
+		require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[i][12])
+	}
+
+	// Test scheduling info for un-partitioned table after split index to regions
+	tk.MustExec("drop table if exists t4_scheduling")
+	tk.MustExec("create table t4_scheduling (id INT, val INT, index idx1(val)) placement policy p1")
+	tk.MustQuery("split table t4_scheduling index idx1 between (0) and (12345) regions 3")
+	re = tk.MustQuery("show table t4_scheduling regions")
+	rows = re.Rows()
+	require.Len(t, rows, 4)
+	require.Len(t, rows[0], 13)
+	for i := range rows {
+		require.Equal(t, "PRIMARY_REGION=\"cn-east-1\" REGIONS=\"cn-east-1,cn-east-2\" SCHEDULE=\"EVEN\"", rows[i][11])
+		require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[i][12])
+	}
+
+	// Test scheduling info for partitioned table after split index to regions
+	tk.MustExec("drop table if exists t5_scheduling")
+	tk.MustExec("create table t5_scheduling (id INT, val INT, index idx1(val)) placement policy p1 partition by range (id) (" +
+		"partition p0 values less than (100) placement policy p2," +
+		"partition p1 values less than (1000)," +
+		"partition p2 values less than (10000)" +
+		")")
+	tk.MustQuery("split table t5_scheduling index idx1 between (0) and (12345) regions 3")
+	re = tk.MustQuery("show table t5_scheduling regions")
+	rows = re.Rows()
+	require.Len(t, rows, 12)
+	require.Len(t, rows[0], 13)
+	for i := range rows {
+		if i < 4 {
+			require.Equal(t, "LEADER_CONSTRAINTS=\"[+region=us-east-1]\" FOLLOWERS=3 FOLLOWER_CONSTRAINTS=\"[+region=us-east-2]\"", rows[i][11])
+		} else {
+			require.Equal(t, "PRIMARY_REGION=\"cn-east-1\" REGIONS=\"cn-east-1,cn-east-2\" SCHEDULE=\"EVEN\"", rows[i][11])
+		}
+		require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[i][12])
+	}
```

Comment:
Add some tests for `show table index regions` ?

Reply:
Done!